pages/_components/demo-snippet.js

```diff
@@ -16,6 +16,14 @@ const hiddenContentRegex = new RegExp('(<!-- docs: start hidden content -->)(.*?
 class DaylightDemoSnippetWrapper extends LitElement {
 	static get properties() {
 		return {
+			/**
+			 * Import the open-first-item-helper to toggle open the first child demo (e.g., dropdown)
+			 */
+			autoOpen: { type: Boolean, attribute: 'auto-open' },
+			/**
+			 * Default values for demo attributes. Formatted as a stringified object.
```

Comment:
```suggestion
			 * Enable this attribute to allow the demo preview to resize to the size of the first child demo (e.g. d2l-button)
```
I think this comment was wrong before